src/BookReader/Mode2UpLit.js

```diff
@@ -337,7 +337,12 @@ export class Mode2UpLit extends LitElement {
         reduce: page.width / wToV(this.computePageWidth(page)),
       }).$container[0];
 
-    pageContainerEl.classList.toggle('BRpage-visible', isVisible);
+    const visibleStatus = pageContainerEl.classList.toggle('BRpage-visible', isVisible);
+    if (visibleStatus) {
+      this.br.trigger('pagevisible', {
+        pageContainerEl,
+      });
+    }
```

Comment:
This is causing pagevisible to fire any time a scroll happens ; maybe only trigger it if it previously didn't have page-visible set? Ditto in 1up.